frontend/src/scenes/session-recordings/player/inspector/PlayerInspectorList.tsx

```diff
@@ -161,12 +159,12 @@ export function PlayerInspectorList(): JSX.Element {
                 .getElementById('PlayerInspectorListMarker')
                 ?.setAttribute('style', `transform: translateY(${offset}px)`)
 
-            if (!syncScrollingPaused && syncScroll) {
+            if (!syncScrollPaused) {
```

Comment:
should syncSrollPaused and listRef be dependencies of the useEffect?

(more a "teach me" question than anything else :)